beacon/pow/build.gradle

```diff
@@ -1,5 +1,6 @@
 dependencies {
   implementation project(':ethereum:pow:api')
+  implementation project(':ethereum:pow:merkletree')
```

Comment:
This winds up not being required if we put move the `DEPOSIT_CONTRACT_TREE_DEPTH` and leave the snapshot schema instance in `DepositTreeSnapshot` rather than in `DepositTree`.